frontend/mulighetsrommet-veileder-flate/src/components/oversikt/Tilgjengelighetsstatus.tsx

```diff
@@ -21,38 +25,41 @@ export function TilgjengelighetsstatusComponent({ status, estimert_ventetid, ste
       <div>
         <div className={styles.tilgjengelighetsstatus}>
           <img src={StatusRod} alt="Rødt ikon som representerer at tiltaksgjennomføringen er stengt" />
-          <div title={`Midlertidig stengt mellom ${formaterDato(stengtFra)} og ${formaterDato(stengtTil)}`}>Midlertidig stengt</div>
+          <div title={`Midlertidig stengt mellom ${formaterDato(stengtFra)} og ${formaterDato(stengtTil)}`}>
+            Midlertidig stengt
+          </div>
         </div>
       </div>
     );
   }
-  if (status === 'Ledig' || !status) {
+  if (status === 'Ledig' || status === 'LEDIG' || !status) {
```

Comment:
Hvorfor må vi håndtere disse to muligetene her (og under)?

Reply:
Verdiene fra Sanity er capitalized, men fra enum i db er all-caps. Jeg gravde ikke for mye i det, men jeg kan se litt på om vi kan sammenstille det på noen god måte før vi sender til frontend.